test/ctia/http/generative/fulltext_search_test.clj

```diff
@@ -319,65 +319,67 @@
                       gen/generate)
             ignore-ks  [:created :groups :id :incident_time :modified :owner :timestamp]]
 
-        ;; to avoid accidentally picking up previous state
-        (store/delete-search incidents-store
-                             {:query "intrusion event 3\\:19187\\:7 incident"
-                              :REALLY_DELETE_ALL_THESE_ENTITIES true}
-                             login nil)
-        (is (-> incidents-store
-                (store/query-string-search
-                 {:query "intrusion event 3\\:19187\\:7 incident"}
-                 login {})
-                :data
-                empty?))
+        (assert (empty?
+                 (-> incidents-store
+                     (store/query-string-search
+                      {:query "intrusion event 3\\:19187\\:7 incident"}
+                      login {})
+                     :data))
+                "The test state is polluted with previous documents.")
 
         (bundle/import-bundle bundle
                               nil    ;; external-key-prefixes
-                              login services)
-        (are [desc query check-fn] (let [res (-> incidents-store
-                                                 (store/query-string-search
-                                                  (merge query {:default_operator "AND"})
-                                                  login {})
-                                                 :data)]
-                                     (testing (str "query: " query)
+                              login
+                              services)
+        (are [desc query check-fn] (let [query-res (store/query-string-search
+                                                    incidents-store
+                                                    (merge query {:default_operator "AND"})
+                                                    login
+                                                    {})
+                                         res (:data  query-res)]
+                                     (testing (format "query:%s\nquery-resi =>\n%s\nbundle =>\n %s"
+                                                      query
+                                                      (with-out-str (clojure.pprint/pprint query-res))
+                                                      (with-out-str (clojure.pprint/pprint bundle))
```

Comment:
Needs a `require` (important because of how CTIA splits tests in CI, each namespace must be loadable independently).